server/repository/src/db_diesel/temperature_chart/temperature_chart_row.rs

```diff
@@ -210,8 +232,19 @@ mod test {
             "SELECT $1 as from_datetime, $2 as to_datetime  UNION  SELECT $3 as from_datetime, $4 as to_datetime"
         };
 
+        let breach_ids_agg = if cfg!(not(feature = "postgres")) {
```

Comment:
:nit swapping the order would mean that you don't need to `not(`

```
let breach_ids_agg = if cfg!(feature = "postgres") {
            "JSON_AGG"
        } else {
            "JSON_GROUP_ARRAY"
        };
```

Reply:
Thanks: cc27fd2b102de6e7c233a6fdd5b16d0dc0b765c2